### PR TITLE
new error mockups

### PR DESCRIPTION
**Pull request by KisKass**

### CODE REVIEW

**Review by inauros — CHANGES_REQUESTED** [review]
I can see mockups with too general information - **An error has occurred, the entered information is not correct**.
But we should consider 2 error types:
1. validation errors (Example from SRS: If any data is entered incorrectly, incorrect data messages are displayed;)
This type of errors is related to the data users entered (Ex.: name must consist of letters only, but user entered numbers, or employee name length should be no more than 40 characters, but user entered 41). Usually, there is a separate validation mechanism and corresponding errors are displaying next to the field with invalid data.
2. system errors (Example from SRS: If an error occurs, then error message is displaying;)
This type of errors is related to some application errors. (Ex. database is not available, or there is inconsistency between UI settings and DB configuration (UI allows to enter name that consists of 40 characters, but DB can store only 35). Such type of errors can happen after users confirm some action (Add, Edit, Delete, etc.), successful data validation (see above) . Usually, such type of errors is displayed in a separate Error pop-up window in UI.

Please, based on what I written above, create 2 types of mockups with error messages.

**Comment by inauros** [comment]
And fix conflicting files as well please.

File: Documentation/SRS.md [hunk 1]
```diff
@@ -3,24 +3,15 @@
...
-
 ![image](https://user-images.githubusercontent.com/83345134/127285174-8b015c12-a1b2-4ff9-a32c-d93a516a4bbb.png "Add employee")
-
-
-To delete an employee user should right-click eployee in the table and press delete button.
-
 ![image](https://user-images.githubusercontent.com/83345134/127285318-fc4847d6-ff49-4f51-863a-19ffafd29391.png "Delete employee")
+![add error](https://user-images.githubusercontent.com/83345134/127639007-3ceae0d9-212e-4714-8424-e515f535e0e3.png)
+![edit error](https://user-images.githubusercontent.com/83345134/127639014-e66fa490-4b0f-487c-8006-6faf4f215ced.png)
+![delete error](https://user-images.githubusercontent.com/83345134/127639021-fb06c230-6baa-466c-a253-d553aeb58a00.png)
```
Comment (IlyaRozumouski) [inline Documentation/SRS.md:22]: i need extra mockup: a validation error for invalid date on the display list of employees 
also as ihar said, there should be two validation errors for adding and edidting employees and three system errors for adding editing or deleting employees

**Review by IlyaRozumouski — COMMENTED** [review]
Also, could u add the "reset" button to reset filter on the employees view list?

**Pull request closed without merging**